client/components/google-my-business/stats-nudge.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+// import Card from 'components/card';
+import DismissibleCard from 'blocks/dismissible-card';
+import SectionHeader from 'components/section-header';
+
+const StatsNudge = ( { translate } ) => {
+	return (
+		<DismissibleCard
+			className="google-my-business__stats-nudge"
+			preferenceName="show-google-my-business-nudge"
+			temporary
```

Comment:
The `temporary` property makes it so dismissing the nudge only lasts for the page session until we figure out how we really want it to be dismissed/presented.